libs/editor/egui_editor/Cargo.toml

```diff
@@ -24,6 +24,7 @@ pulldown-cmark = { version = "0.9.2", default-features = false }
 reqwest = { version = "0.11", features = ["blocking"] }
 unicode-segmentation = "1.10.0"
 rand = "0.8.5"
+linkify = "0.10.0"
```

Comment:
https://github.com/robinst/linkify: 164 stars, last commit 24 Jun 2023. "Your reaction might be: Do I need a library for this? Why not a regex?. Let's look at a few cases..." -their readme